bot/exts/utilities/epoch.py

```diff
@@ -86,7 +86,10 @@ async def epoch(self, ctx: commands.Context, *, date_time: DateString = None) ->
         view = TimestampMenuView(ctx, self._format_dates(date_time), epoch)
         original = await ctx.send(f"`{epoch}`", view=view)
         await view.wait()  # wait until expiration before removing the dropdown
-        await original.edit(view=None)
+        try:
+            await original.edit(view=None)
+        except discord.NotFound:  # disregard the error message if the message is deleled
+            pass
```

Comment:
Usually we'd use [`contextlib.suppress`](https://docs.python.org/3/library/contextlib.html#contextlib.suppress) here, but that's more of a nitpick and doesn't really matter.